src/Kestrel.Core/Internal/Http/HttpResponseStream.cs

```diff
@@ -145,25 +146,23 @@ public void Abort()
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Do you need the attribute for this method? It looks pretty small

Reply:
Unfortunately yes. At least on `linux-x64` where I checked the dasm.

Reply:
What's the reason for not inlining? Unprofitable?

Reply:
Yeah `unprofitable inline`.